### ISSUE TITLE
aaNotation improvement

### ISSUE THREAD

**Issue by KdudeDev** [KdudeDev/InfiniteMath#39]
I'd like to improve aaNotation so that you can decide how many letters can show (a, aa, aaa, etc.) instead of just 2. It would also only show a letter once the previous letter has reached z, so you'd get up to 1z before getting 1aa, 1zz before 1aaa, etc. Instead of 1azz, 1aaa.

Been having some problems with this however, Got it sort of functional but it would skip over z (go from 1y to 1aa). In my attempts to fix I created a new problem of having letters get added on too early, 1y, 1az, 1aa. Lots of other issues but these were the 2 main ones.

tldr. rewrite the aaNotation function to support a chosen amount of letters instead of just 2 and only show letters once needed.
